nodes/core/io/21-httpin.js

```diff
@@ -134,6 +159,16 @@ module.exports = function(RED) {
                         }
                         msg.res.set('content-length', len);
                     }
+                    
+                    // Deliberately setting AFTER content-length has been calculated as these fields will be deleted by middleware!
```

Comment:
Potential bug if the middleware forgets to call <code>delete res._metrics</code> . My middleware is currently OK but maybe a maintenance programmer might not realise that the content-length is wrong so long as res._metrics is still attached to res! How could I link the two together? Is there a convention?